src/compiler/gcc.rs

```diff
@@ -247,6 +247,7 @@ where
     let mut extra_hash_files = vec![];
     let mut compilation = false;
     let mut multiple_input = false;
+    let mut multiple_input_files: Vec<String> = vec![];
```

Comment:
```suggestion
    let mut multiple_input_files = Vec::<String>::new();
```

Reply:
Changed. Although 9 lines below there is similar preexisting code with the previous syntax:
```rust
let mut xclangs: Vec<OsString> = vec![];
```